### ISSUE TITLE
When switching between minerals on Projection Availability Card data is not getting inputted into dialog.

### ISSUE THREAD

**Issue by DanielLyleWilliamWarren** [micronutrientsupport/micronutrient-support-tool#673]
## Steps to Reproduce
```
- Select Malawi + Zinc from side nav
- Click 'Explore projections to 2050 tab' 
- Click 'Expand Card' then close the card
- Select Magnesium from mineral dropdown
- Click 'Expand Card' and new data is not being inputted.
```

AFAIK this issue is unique to this card

**Comment by DanielLyleWilliamWarren** [comment]
Issue resolved by dw in MR #680

**Comment by Fuhji** [comment]
closed in #685
